src/components.js

```diff
@@ -826,14 +827,14 @@ const components = [
     name: 'Fixed',
     type: 'div',
     props: {},
-    style: props => ({
+    style: props => Object.assign({
       position: 'fixed',
       top: props.top ? 0 : null,
       right: props.right ? 0 : null,
       bottom: props.bottom ? 0 : null,
       left: props.left ? 0 : null,
       zIndex: props.z
-    }),
+    }, center(props)),
```

Comment:
I have thought about splitting the components module up, but I think that's outside the scope of what you initially proposed in this PR and would conflict with some of the other changes I have planned